scripts/run_tests.sh

```diff
@@ -66,7 +66,7 @@ while :; do
         kill $bg_pid
         exit 127
     fi
-    if grep -q "The Jupyter Notebook is running at:" $OUTPUT_FILE; then
+    if grep -q "Jupyter Notebook.*is running at:" $OUTPUT_FILE; then
```

Comment:
Different test to see if it's running! This threw me for a bit of a loop for an embarrassing amount of time.